### PR TITLE
Pypi url

### PR DESCRIPTION
**Pull request by jburel**
This PR fixes a problem noticed by @pwalczysko see https://github.com/ome/omeroweb-install/pull/33#pullrequestreview-62385374

I have also updated few links to use ``:pypi:``

### CODE REVIEW

**Comment by pwalczysko** [comment]
```
[exec] Warning, treated as error:
     [exec] /home/travis/build/openmicroscopy/ome-documentation/omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst:227: WARNING: Duplicate explicit target name: "pypi".
```

**Comment by jburel** [comment]
I will have to manually update those files
They are updated via autogen!!!

**Comment by jburel** [comment]
@pwalczysko travis is  happy now

File: contributing/tools-maintenance.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Development tools: maintenance
 ==============================
 
 Most of the development tools used internally by the OME project are
-Python-based and hosted on the :pypi:`Python Package Index <>`. Their source
+Python-based and hosted on `PyPI`_. Their source
```
Comment (sbesson) [inline contributing/tools-maintenance.rst]: The backquotes are unnecessary here and below, using `hosted on PyPI_.` should work.

**Review by sbesson — CHANGES_REQUESTED** [review]
Want to edit the top-level README as well to include `PyPI` in the list of global substitutions?

File: contributing/tools-maintenance.rst (hunk 1 above)
Comment (jburel) [reply contributing/tools-maintenance.rst]: other links could be updated too
but I won't fix that in this PR

**Comment by jburel** [comment]
@sbesson done

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-docs#841](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/841/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/841/consoleText) for more details.
Possible conflicts:
  - PR #1756 hflynn 'Whatsnew for users & sysadmins & CLI signposting'
    - omero/sysadmins/whatsnew.rst

**Comment by pwalczysko** [comment]
Checked all the PyPI links as listed in the diff one-by-one.
The two requests by @sbesson (quotes and README addition) seem to be satisfied.
The other changes are as expected.

Ready to merge FMPOV.

**Review by sbesson — APPROVED** [review]

**Pull request merged**